devito/operator/operator.py

```diff
@@ -7,6 +7,7 @@
 from tempfile import gettempdir
 
 from sympy import sympify
+from sympy import Basic as SympyBasic
```

Comment:
ultra nitpick: for homogeneity, I think just `import sympy` and then `sympy.Basic` where necessary

Reply:
Sure